### PR TITLE
Parse coverageLocations from the string parameter

### PR DESCRIPTION
**Pull request by MartinNuc**
Fixes https://github.com/paambaati/codeclimate-action/issues/153

### CODE REVIEW

**Comment by paambaati** [comment]
@MartinNuc Thank you so much for the PR! Can you link me to test Action runs with this PR? I’d like to see this work for both with and without coverage locations defined.

**Comment by MartinNuc** [comment]
@paambaati sure, let me fix prettier errors and prepare showcase repo :-) brb

**Comment by MartinNuc** [comment]
I prepared testing repo here: https://github.com/MartinNuc/coverage-ga-test/actions/runs/85624708

File: README.md [hunk 1]
```diff
@@ -43,9 +43,47 @@ steps:
...
+```
+
+```yaml
+steps:
+  - name: Test & publish code coverage
+    uses: paambaati/codeclimate-action@v2.5.7
+    env:
+      CC_TEST_REPORTER_ID: ${{secrets.CC_TEST_REPORTER_ID}}
+    with:
+      # coverageCommand: yarn coverage - this line is actually not needed because `yarn coverage` is by default
```
Comment (paambaati) [inline README.md:83]: @MartinNuc Is this documented?

Comment (MartinNuc) [reply README.md:83]: Yes the default value is documented in the table here https://github.com/paambaati/codeclimate-action#inputs

**Pull request merged**